classes/class.tx_caretakerinstance_SecurityManager.php

```diff
@@ -103,27 +103,25 @@ public function __construct(tx_caretakerinstance_ICryptoManager $cryptoManager)
      * - Encrypted data signature
      *
      * @param tx_caretakerinstance_CommandRequest $commandRequest
+     * @throws Exception
      * @return bool
      */
     public function validateRequest(tx_caretakerinstance_CommandRequest $commandRequest)
     {
         $sessionToken = $commandRequest->getSessionToken();
         $timestamp = $this->cryptoManager->verifySessionToken($sessionToken, $this->privateKey);
         if ((time() - $timestamp) > $this->sessionTokenExpiration) {
-            // Session token expired
-            return false;
+            throw new Exception('Session token expired');
```

Comment:
I would prefer to throw specific exceptions like throw new SessionTokenExpired('Session token expired', 123456789)